lib/mutant/meta/example/dsl.rb

```diff
@@ -9,18 +8,20 @@ class DSL
         # Run DSL on block
         #
         # @return [Example]
-        def self.run(file, block)
+        def self.call(file, block)
```

Comment:
`.call` is the better convention. As Rubies own blocks / procs / lambdas follow these.